docs/src/rules/arrow-parens.md

```diff
@@ -133,7 +133,7 @@ var b = 0;
 if ((a) => b) {
  console.log('truthy value returned');
 } else {
- console.log('falsey value returned');
+ console.log('falsy value returned');
```

Comment:
`falsy` can sometimes be written as `falsey` too according to [MDN](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

Reply:
I changed it to falsy just for consistency with the rest of the codebase, which uses falsy over falsey.

Reply:
If that's the case you can resolve my change. Thanks